.changeset/config.json

```
@@ -0,0 +1,24 @@
+{
+  "$schema": "https://unpkg.com/@changesets/config@2.3.0/schema.json",
+  "changelog": ["@changesets/changelog-github", { "repo": "vuestorefront/storefront-ui" }],
+  "commit": false,
+  "fixed": [],
+  "linked": [],
+  "access": "public",
+  "baseBranch": "origin/master",
+  "updateInternalDependencies": "minor",
```

[Shouldn't this be `patch`](https://github.com/changesets/changesets/blob/main/docs/config-file-options.md)?
```suggestion
  "updateInternalDependencies": "patch",
```

---

TBH i wasnt sure how it should work